.gitignore

```diff
@@ -1,6 +1,10 @@
 node_modules/
 tmp/
+build/
```

Comment:
Not sure where the `build/` comes from.
@akreuzkamp, why do we want this in `.gitignore`?